docs/reference/commandline/dockerd.md

```diff
@@ -1323,6 +1324,7 @@ This is a full example of the allowed configuration options on Linux:
 	"cluster-advertise": "",
 	"max-concurrent-downloads": 3,
 	"max-concurrent-uploads": 5,
+    "max-download-attempts": 5,
```

Comment:
Looks like this is using spaces for indentation, and other lines use tabs; I'll push a fix to this branch